internal/connection/connection.go

```diff
@@ -29,23 +31,67 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+const (
+	// The duration after which a gRPC connection is closed due to inactivity
+	IDLE_TIMEOUT = time.Minute * 5
```

Comment:
Call this `IdleTimeout`, or, if this is internal to this package, `idleTimeout`. Capitalization like `IDLE_TIMEOUT` is really not common in Go.